packages/runtime-dom/src/components/Transition.ts

```diff
@@ -163,9 +163,13 @@ export function resolveTransitionProps(
       // the transition starts. This is applied for enter transition as well
       // so that it accounts for `visibility: hidden` cases.
       const cachedTransition = (el as HTMLElement).style.transitionProperty
-      ;(el as HTMLElement).style.transitionProperty = 'none'
+      if (cachedTransition) {
```

Comment:
This way will reopen #2593, maybe a better way is needed to fixed both of issues.

Reply:
After add this code:
``` javascript
;(el as HTMLElement).style.transitionProperty = 'none'
```

`hasCSSTransform` will be `false`
https://github.com/vuejs/vue-next/blob/d067fb2dbb82d9a653308baec40bb7f42322b705/packages/runtime-dom/src/components/TransitionGroup.ts#L60-L68

beacuse `hasTransform` will be `false`
https://github.com/vuejs/vue-next/blob/d067fb2dbb82d9a653308baec40bb7f42322b705/packages/runtime-dom/src/components/Transition.ts#L344-L346

if change the RegEx to:
``` javascript
/\b(transform|all|none)(,|$)/.test(styles[TRANSITION + 'Property']) 
```
will works fine.but this is bad solution.
I didn't figure out a better solution yet.

Reply:
I'm also trying to find a new solution.

Reply:
I'm not sure,  but maybe a more acceptable solution #2716

Reply:
> I'm not sure, but maybe a more acceptable solution #2716

Looks better.I will close this PR.